pkg/consts/qrm.go

```diff
@@ -19,4 +19,16 @@ package consts
 const (
 	// KubeletQoSResourceManagerCheckpoint is the name of the checkpoint file for kubelet QoS resource manager
 	KubeletQoSResourceManagerCheckpoint = "kubelet_qrm_checkpoint"
+
+	// CPUResourcePluginPolicyNameDynamic is the name of the dynamic policy.
+	CPUResourcePluginPolicyNameDynamic = "dynamic"
```

Comment:
would it be better to put cpu related consts under cpu package?

Reply:
done.